Resources/Prototypes/XenoArch/effects.yml

```
@@ -588,33 +588,33 @@
 - type: entity
   id: XenoArtifactThrowThingsAround
   parent: BaseXenoArtifactEffect
-  description: Minor implosion
+  description: Environmental disruption # Moffstation - Obfuscate xenoarcheology
   components:
   - type: XAEThrowThingsAround
 
 - type: entity
   id: XenoArtifactColdWave
   parent: BaseXenoArtifactEffect
-  description: Cools down surrounding gas
+  description: Energy consumption # Moffstation - Obfuscate xenoarcheology
```

```suggestion
  description: Endothermic action # Moffstation - Obfuscate xenoarcheology
```
I like the big science word. Maybe the `-therm-` would be kinda misleading, but, "mislead" is just another pronunciation of "obfuscate", right? :^)